ios/RNAudioStream.m

```diff
@@ -25,7 +25,7 @@ - (dispatch_queue_t)methodQueue {
     if (url && url.scheme && url.host) {
         [UIApplication sharedApplication].idleTimerDisabled = YES;
 
-        _streamPlayer = [[MPMoviePlayerController alloc] initWithContentURL: url];
+        _streamPlayer = [AVPlayer playerWithURL: url];
```

Comment:
I'm not entirely sure this is the best way to do this. Ideally we'd separate out `play` from `init` but this is experimental, still.